homeassistant/components/nut/const.py

```diff
@@ -55,6 +55,7 @@
         key="ups.status",
         name="Status Data",
         icon="mdi:information-outline",
+        entity_category=ENTITY_CATEGORY_DIAGNOSTIC,
```

Comment:
Status is probably the main sensor for NUT. May be is better to not assign any category (as status display that is based on this)?

Reply:
the reasoning here was that you have status and status_data. Both essentially show the same but status is more user friendly given it will say `Online` rather than `OL`. Wouldn't then one sensor be enough as the main sensor rather than having both?
![image](https://user-images.githubusercontent.com/22405505/139951802-1fe8ac07-3167-4828-bc0d-fd7eba654b91.png)

Reply:
That's true, but `Status` is just a conversion of `Status Data` and is not provided directly by the device. Just point of view...